src/image/gtk.rs

```diff
@@ -248,39 +248,60 @@ impl Deref for IconLabel {
     }
 }
 
-#[derive(Clone, Debug)]
 #[cfg(feature = "music")]
 pub struct IconPrefixedLabel {
     label: Label,
+    overflow: Option<crate::gtk_helpers::OverflowLabel>,
```

Comment:
Would rather this didn't use the fully qualified path, happy to import:

```suggestion
    overflow: Option<OverflowLabel>,
```

Same in one or two cases below